src/Integration/Binding/BindingController.cs

```diff
@@ -169,6 +172,8 @@ private void OnBindingFinished(ProjectInformation projectInformation, bool isFin
             }
             else
             {
+                this.host.VisualStateManager.ClearBoundProject();
```

Comment:
Should not be needed since it should start in unbound state as soon as the workflow have started.